CHANGELOG.md

```diff
@@ -10,31 +10,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [12.0.0] - 2022-03-10
 
-### Fixed
-- Filter cells with formulas using values not source [#5455](https://github.com/handsontable/handsontable/issues/5455)
-- Fixed not working docs building script on Windows [#8677](https://github.com/handsontable/handsontable/issues/8677)
-- Fixed an issue with BottomOverlay position for RTL [#8761](https://github.com/handsontable/handsontable/issues/8761)
-- Fixed an issue with TopOverlay position for RTL [#8764](https://github.com/handsontable/handsontable/issues/8764)
-- Fixed an issue with numeric cell formatting for RTL [#8791](https://github.com/handsontable/handsontable/issues/8791)
-- Fixed an issue with htAutocompleteArrow position for RTL [#8795](https://github.com/handsontable/handsontable/issues/8795)
-- Fixed an issue with checkboxes positions. [#8797](https://github.com/handsontable/handsontable/issues/8797)
-- Fixed an issue with opening position of the dropdown (autocomplete) editor. [#8802](https://github.com/handsontable/handsontable/issues/8802)
-- Fixed an issue with time cell rendering for RTL [#8805](https://github.com/handsontable/handsontable/issues/8805)
-- Fixed an issue with context menu in RTL mode [#8825](https://github.com/handsontable/handsontable/issues/8825)
-- Fixed an issue with filters menu UX for RTL [#8828](https://github.com/handsontable/handsontable/issues/8828)
-- Wrong TypeScript definition in the BasePlugin class [#9175](https://github.com/handsontable/handsontable/issues/9175)
-- Fixed an issue with the code examples' deployment workflow, where not all example directories were deployed to gh-pages. [#9182](https://github.com/handsontable/handsontable/issues/9182)
-- *React:* Fixed an issue with registering modules for React wrapper [#9140](https://github.com/handsontable/handsontable/issues/9140)
+### Added
+- **Breaking change**: Added `ShortcutManager`, a new API for customizing keyboard shortcuts. [#8942](https://github.com/handsontable/handsontable/issues/8942)
```

Comment:
It looks weird to see **Braking change** here, especially since this point doesn't say what the breaking change is.

Please remove the word **Braking change** from this point and make a new point under "Changed" heading about the breaking change.